mobile/src/main/res/values/strings.xml

```diff
@@ -146,6 +147,15 @@
     <string name="nfc_action_current_color">Current color</string>
     <string name="nfc_action_to_sitemap_page">Navigate to sitemap page</string>
 
+    <!-- Geofencing -->
+    <string name="geofence_dialog_new">New Geofence</string>
+    <string name="geofence_label">Label</string>
+    <string name="geofence_name">Name</string>
+    <string name="geofence_coordinates">Coord.</string>
```

Comment:
looks bad on my phone if written out

Reply:
In abbreviated form it'll likely look bad on someone else's phone. This sounds like it could/should be solved by a better placement of controls instead. Can you provide a screenshot of the dialog?